en/mkdocs.yml

```diff
@@ -176,7 +176,7 @@ nav:
         - Deploying APIs in API Gateway vs API Microgateway: deploy/deploying-apis-in-api-gateway-vs-api-microgateway.md
         - Deploy API:
             - Deploy an API: deploy/deploy-api/deploy-an-api.md
-            - Grouping APIs with Labels: deploy/deploy-api/grouping-apis-with-labels.md
+            - Grouping APIs with Virtual Hosts: deploy/deploy-api/grouping-apis-with-vhosts.md
```

Comment:
This needs to be changed based on the changes done in the main file.
In addition, if we are removing the "Grouping APIs with Labels" feature it needs to be mentioned in the About this release page, else there should be a redirection for the "Grouping APIs with Labels" page.